### ISSUE TITLE
accessibility problem

### ISSUE THREAD

**Issue by danielbotn** [dozoisch/react-google-recaptcha#164]
react-google-recaptcha version: 1.0.5
react-async-script version: don't know
.react version:  16.8.1

Our accessibility software is flagging a violation that iframe is missing a title attribute.

Is there any way to provide the frame with the attribute title and add a description of the content in the title?

**Comment by dozoisch** [comment]
@danielbotn they don't see to provide a way to touch the iframe from here https://developers.google.com/recaptcha/docs/display#configuration≈ 

Unless I'm missing something
